scipy/stats/_continuous_distns.py

```diff
@@ -7040,11 +7040,25 @@ def _stats(self, df):
     def _entropy(self, df):
         if df == np.inf:
             return norm._entropy()
+
         half = df/2
         half1 = (df + 1)/2
-        return (half1*(sc.digamma(half1) - sc.digamma(half))
-                + np.log(np.sqrt(df)*sc.beta(half, 0.5)))
 
+        # _lazywhere complains if the functions don't have inputs
+        def regular(df):
+            return (half1*(sc.digamma(half1) - sc.digamma(half))
+                    + np.log(np.sqrt(df)*sc.beta(half, 0.5)))
+
+        def asymptotic(df):
+            # gammaln(x) ~ x * ln(x) - x - 0.5 * ln(x) + 0.5 * ln(2 * pi)
+            # psi(x) ~ ln(x) - 1 / (2 * x)
+            # B(x, y) ~ gamma(x) * y ** (-x)
+            # (x / 2) * (psi(x + 1) - psi(x)) ~ 0.5
```

Comment:
```suggestion
```

Looks like this was left over from the old formula.

Reply:
That formula explains where the 0.5 came from, so we should keep it.

Reply:
The formula Wolfram Alpha is giving you doesn't appear to follow from these facts alone. For example, the appearance of `x*-2`, `x**-3`, etc... is not explained by the facts above; the disappearance of `log(df)` terms in the result is also not explained. So I'd suggest removing this and indicating where this formula actually came from.